scripts/cibuildpkg.py

```diff
@@ -117,6 +117,7 @@ class Package:
     source_dir: str = ""
     source_filename: str = ""
     source_strip_components: int = 1
+    lgpl_compatible: bool = True  # Compatible with the LGPLv3 license
```

Comment:
The naming seems dodgy. What you're actually describing is whether the code is non-GPL.